index.html

```diff
@@ -16,7 +16,11 @@ <h2>Our Advice for Newbies!</h2>
     </header>
 
     <section>
+<<<<<<< HEAD
+      <a href="advice/gillian-hanlon-advice.html">Gillian's General Guidance</a><br>
+=======
```

Comment:
Remove this

Reply:
Just the <<<<<HEAD and the ====== 

not your link! :)

Reply:
Hi Ann. I removed the info from my code editor. I don't know how its still showing up on Github.

Reply:
Ok so you'll need to save the file, add and commit the change in your Terminal, and then push up you branch again. @GMHanlon 

Then the changes should make it here.